src/gr11review/part2/Utility.java

```diff
@@ -1,9 +1,352 @@
+
+                    longestWord = tempString;
+                    length = tempString.length();
+
+                }
+
+        }
+
+        myReader.close();
+
+        return longestWord;
+
+   }
+  
+    /**
+     * File IO 2
+     * Returns the alphabetically first word in a text file
+     * @author P. Tran
+     */
+    public static String alphaWord(String filenametxt)throws IOException {
+
+      String alphabetical = "";
+
+      try {
+        BufferedReader keyboard = new BufferedReader(new FileReader(filenametxt));
+      
+        String fileLine = "";
+        alphabetical = keyboard.readLine();
+
+        while(fileLine != "null") {
+          fileLine = keyboard.readLine();
+          if (fileLine != "null" && alphabetical.compareTo(fileLine) > 0) {
+            alphabetical = fileLine;
+          }
+        }
+
+        keyboard.close();
+
+       } catch(Exception e) {
+         System.out.println("Error.");
+       }
+
+       return alphabetical;
+     }
+
+     /**
+      *Array 1 One Dimensional
+      *For each multiple of 10 in the given array, 
+      *change all the values following it to be that multiple of
+      *10, until encountering another multiple of 10.
+      *So {2, 10, 3, 4, 20, 5} yields {2, 10, 10, 10, 20, 20}.
+      *@author A. Wong
+      */
+
+      public static int[] tenRun(int[] nums){
+
+        int currentMultiple = 0;
+        boolean hasReachedFirst = false;
+
+        //Checks if the number is divisible by 10 without remainders
+        //Replaces all following numbers with multiple of 10
+        //If there is a new multiple of 10, it takes that number instead
+        for (int counter = 0; counter < nums.length; counter++){
+
+            if (nums[counter] % 10 == 0){
+
+                currentMultiple = nums[counter];
+                hasReachedFirst = true;
+
+            }
+            else if (hasReachedFirst){
+
+                nums[counter] = currentMultiple;
+
+
+            }
+
+
+        }
+
+        return nums;
+
+      }
+  
+      /**
+        * Array 2
+        * All "alone" elements take on the greater of the two values to the left and right of it
+        * @author P. Tran
+        */
+      public static int[] notAlone(int[] nums, int value) {
+
+        //Checks if each element of the array with the value is alone
+        for (int i = 0; i < nums.length; i++) {
+          if (nums[i] == value && i > 0 && i < nums.length - 1) {
+            if (nums[i] != nums[i - 1] && nums[i] != nums[i + 1]) {
+              nums[i] = Math.max(nums[i - 1], nums[i + 1]);
+            }
+          }
+        }
+
+        return nums;
+      }
+
+
+      /**
+       * Array 3 One Dimensional Two Loops
+       * Checks to see if the outer array
+       * Has the numbers of the inner array
+       * Assuming both arrays are sorted by number size
+       * @author A. Wong
+       */
+      public static boolean linearIn(int[] outer, int[] inner){
```

Comment:
@WearyWasTaken you need to tighten up your whitespacing in your methods.  excessive space makes it less readable